resources/definitions/ultimaker2.def.json

```diff
@@ -95,7 +95,6 @@
         "machine_max_feedrate_y": { "default_value": 300 },
         "machine_max_feedrate_z": { "default_value": 40 },
         "machine_name": { "default_value": "Ultimaker 2" },
-        "machine_nozzle_head_distance": { "default_value": 3 },
         "machine_nozzle_tip_outer_diameter": { "default_value": 1 },
         "machine_start_gcode": { "value": "\"G0 F3000 Y50 ;avoid prime blob\"  if machine_gcode_flavor == \"UltiGCode\" else \"G21 ;metric values\\nG90 ;absolute positioning\\nM82 ;set extruder to absolute mode\\nM107 ;start with the fan off\\nG28 Z0 ;move Z to bottom endstops\\nG28 X0 Y0 ;move X/Y to endstops\\nG1 X15 Y0 F4000 ;move X/Y to front of printer\\nG1 Z15.0 F9000 ;move the platform to 15mm\\nG92 E0 ;zero the extruded length\\nG1 F200 E10 ;extrude 10 mm of feed stock\\nG92 E0 ;zero the extruded length again\\nG1 Y50 F9000\\n;Put printing message on LCD screen\\nM117 Printing...\"" },
         "machine_use_extruder_offset_to_offset_coords": { "default_value": true },
```

Comment:
This seems to be a bug in the bot. The relevant strings it wants to replace (and for some reason remove a linebreak, which is NOK) are (otherwise) exactly the same.